apps/system/js/notifications.js

```diff
@@ -369,6 +371,14 @@ var NotificationScreen = {
     return date;
   },
 
+  /**
+   * Check if notification is from system and marked silent in icon url.
+   */
+  isSilentNotification: function ns_isSilentNotication(detail) {
+    return detail.manifestURL === this.SYSTEM_MANIFEST_URL &&
+      detail.icon && detail.icon.indexOf('silent=1') !== -1;
```

Comment:
silent=1 for an icon url really? Is there no way to make this some kind of detail property?